lib/live_view_native_swift_ui/modifiers/controls_and_indicators/button_style.ex

```diff
@@ -10,4 +10,7 @@ defmodule LiveViewNativeSwiftUi.Modifiers.ButtonStyle do
       plain
     )a)
   end
+
+  def params(style) when is_atom(style), do: [style: style]
```

Comment:
It's more verbose, but I would recommend guarding against booleans and `nil` values which are considered atoms in Elixir / Erlang:

```suggestion
  def params(style) when is_atom(style) and not is_boolean(style) and not is_nil(style), do: [style: style]
```